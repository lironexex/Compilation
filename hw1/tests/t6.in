// Integers - DEC_INT, HEX_INT:
var num1 = 1 + 0010 + 123 * 0xABC1DEF + 0x0
var num2 = 0xDeF -100 + 0xabc101c
0x0000000000101
0000000000009
00xDef
0Xabc
