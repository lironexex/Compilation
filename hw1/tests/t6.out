1 COMMENT 1
2 VAR var
2 ID num1
2 ASSIGN =
2 DEC_INT 1
2 BINOP +
2 DEC_INT 10
2 BINOP +
2 DEC_INT 123
2 BINOP *
2 HEX_INT 180100591
2 BINOP +
2 HEX_INT 0
3 VAR var
3 ID num2
3 ASSIGN =
3 HEX_INT 3567
3 BINOP -
3 DEC_INT 100
3 BINOP +
3 HEX_INT 180097052
4 HEX_INT 257
5 DEC_INT 9
6 DEC_INT 0
6 ID xDef
7 DEC_INT 0
7 ID Xabc
